### DISCUSSION TITLE
At startup Timer tab be active instead of Stats

### DISCUSSION THREAD

**Discussion by pedroduarte0** [focustimerhq/FocusTimer#569] [Ideas]
Hi,

(Great application BTW, it's my preferred pomodoro method application!)

Is there any option where allows Pomodoro UI to show the Timer tab active instead of the Stats?
Personally I am not interested on statistics but more important it forces me do to an unnecessary action of clicking the Timer tab before I start using the application.

Best regards

**Comment by kamilprusko** [comment]
It's changed in `main` branch / 1.0-alpha.

Reason behind this was that with extension enabled, there's more sense to use stats than the timer.
